### PR TITLE
HOLD: Adding robo - redo after https://github.com/United-Philanthropy-Forum/km-starter-kit/pull/25

### PR DESCRIPTION
**Pull request by mariacha**
### For the PR creator:
Check off the following boxes before asking someone to review this PR.

- [x] If this PR changes the `composer.json` file, is there accompanying text in the `README.md` file explaining that change?
- [x] Update the "How to test this branch" instructions below by replacing `adding-robo` with the name of this branch, and 
`robotest` with an example project name.

### What this PR does:
- [ ] Adds robo. It's an older version because we're locked at drush 8x, but it still works.

### How to test this branch:
- [ ] If you haven't already, run the [One time setup steps](https://github.com/United-Philanthropy-Forum/km-starter-kit/wiki/How-to-test-changes-to-this-starter-kit#one-time)
- [ ] Run this:

```yaml
COMPOSER_MEMORY_LIMIT=-1 terminus build:project:create --team='United Philanthropy Forum' --org='United-Philanthropy-Forum' --visibility='private' --stability=dev "united-philanthropy-forum/km-starter-kit:dev-adding-robo" robotest
```

- [ ] Validate that a new site exists at https://dev-robotest.pantheonsite.io
- [ ] Validate the repo exists at https://github.com/United-Philanthropy-Forum/robotest
- [ ] Update your .env.dist to contain the `robotest` in the "TERMINUS_SITE" section. Make sure you can run a `robo install` from the `dev` environment.
- [ ] Delete the test environment when you're done, by running `terminus build:env:obliterate robotest`
- [ ] Once this PR is merged, cut a new tag so the `create-project` commands will get the latest and greatest.

## TODOS:
- [ ] We'll want to document how to use `robo install` somewhere. Probably here: https://github.com/United-Philanthropy-Forum/km-collaborative/wiki/Local-development#pulling-down-pantheon-database

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +117,48 @@ This code belongs in this project only. It should only need to be updated if cha
...
+
+* The "compile-code" script is added, which compiles the kmc base theme into css/js instead of sass.
+
+* The "post-drupal-scaffold-cmd" sets the permission levels for a couple of circle-build files to 755, so it can run on circle as a bash script. These files are also customized, along with the '.circleci/config.yml' file, to belong to the site-specific repository, instead of being pulled from the example drops repo.
+
+* The "km-collab-scaffold" was added to the "extras" section. These are configurations passed to the "scripts/composer/ScriptUpdater.php" script.
+
+* The option to allow upstream modules to apply patches was enabled via the "enable-patching" section. This way, the [KM Collaboration profile](https://github.com/United-Philanthropy-Forum/km-collaborative) can be responsible for things like drupal core patches.
+
+* The patchLevel was set to -p2 for the drupal/core package, which keeps a rouge "b" folder from showing up in your web/core folder in cases
```
Comment (unclegcb) [inline README.md:159]:
```suggestion
* The patchLevel was set to -p2 for the drupal/core package, which keeps a rogue "b" folder from showing up in your web/core folder in cases
```

**Review by unclegcb — APPROVED** [review]
@mariacha any reason not to merge this now?

**Comment by unclegcb** [comment]
Okay I'm looking at this again and it's real old. We probably need to re-do this?

**Comment by mariacha** [comment]
Oh yeah, probably going to just drop this since Robo-drupal is handy for sites we work on but we don't want to impose it on other sites. Closing and dropping.

**Pull request closed without merging**